gdk4/src/gl_texture.rs

```diff
@@ -38,4 +41,14 @@ impl GLTexture {
             Box::into_raw(released_func) as glib::ffi::gpointer,
         ))
     }
+
+    #[cfg(feature = "v4_12")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v4_12")))]
+    // rustdoc-stripper-ignore-next
+    /// Creates a new builder-pattern struct instance to construct [`GLTexture`] objects.
+    ///
+    /// This method returns an instance of [`GLTextureBuilder`](crate::builders::GLTextureBuilder) which can be used to create [`GLTexture`] objects.
+    pub fn builder() -> GLTextureBuilder {
```

Comment:
You might also want to fix the API then. It's not really a Rust-style builder right now because you can't chain the methods (they're setters)

Reply:
I have an open issue to handle the GLTextureBuilder API. Will leave that to later as it is not really blocking anything

Reply:
Makes sense